### PR TITLE
Add style for Hero block

### PR DESCRIPTION
**Pull request by nicolasbarbe**
Fix #[10](https://github.com/hlxsites/rolex-event/issues/10)

Test URLs:
- Before: https://hero--rolex-event--hlxsites.hlx.page/transfers
- After: https://main--rolex-event--hlxsites.hlx.page/transfers

Content has been changed to include a picture in the hero block.

### CODE REVIEW

File: blocks/hero/hero.css [hunk 1]
```diff
@@ -10,31 +10,48 @@ main .hero-container {
...
-  padding: 32px;
   min-height: 300px;
 }
 
 main .hero h1 {
   max-width: 1200px;
   margin-left: auto;
   margin-right: auto;
-  color: white;
+
```
Comment (solaris007) [inline blocks/hero/hero.css:20]: extraneous line

File: blocks/hero/hero.css [hunk 2]
```diff
@@ -10,31 +10,48 @@ main .hero-container {
...
   margin-right: auto;
-  color: white;
+  
 }
 
 
 main .hero picture {
-  position: absolute;
   z-index: -1;
+  height:300px;
```
Comment (solaris007) [inline blocks/hero/hero.css:26]: space after column - does your IDE have eslint enabled?

Comment (nicolasbarbe) [reply blocks/hero/hero.css]: I have enabled it but it didn't complained, should I configure some specific rules somewhere?

File: blocks/hero/hero.css (hunk 1 above)
Comment (nicolasbarbe) [reply blocks/hero/hero.css]: Fixed

File: blocks/hero/hero.css (hunk 2 above)
Comment (solaris007) [reply blocks/hero/hero.css]: you could add `declaration-colon-space-after` to `.stylelintrc.json`

Comment (nicolasbarbe) [reply blocks/hero/hero.css]: Just pushed another commit to set such rule.

**Comment by solaris007** [comment]
@ramboz what's the philosophy, should only grid-layouting be used and no more position relative / display flex, etc.?

**Comment by ramboz** [comment]
@nicolasbarbe Do you plan to do some more work on this PR or are we good to merge here?

@solaris007 No strong opinion. I'd say we use whatever works best for the dev. As long as we stay fluid with the responsive design, whether you use grid, flex, or something else doesn't really matter

**Review by ramboz — APPROVED** [review]
Looks good overall to me

**Review by solaris007 — APPROVED** [review]

**Comment by nicolasbarbe** [comment]
Thanks @ramboz , @solaris007 , PR has been merged

**Pull request merged**